wlauto/workloads/skypeecho/uiauto/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+class_dir=bin/classes/com/arm/wlauto/uiauto
+base_class=`python -c "import os, wlauto; print os.path.join(os.path.dirname(wlauto.__file__), 'common', 'android', '*.class')"`
+mkdir -p $class_dir
+cp $base_class $class_dir
+
+ant build
+
+exit_code=$?
```

Comment:
Yes it does. Ideally this should be in the template file that generates build.sh, so that all new workloads pick it up. Existing ones can be changed on demand.